pkg/infoschema/interface.go

```diff
@@ -30,7 +30,14 @@ type InfoSchema interface {
 	context.MetaOnlyInfoSchema
 	TableByName(ctx stdctx.Context, schema, table ast.CIStr) (table.Table, error)
 	TableByID(ctx stdctx.Context, id int64) (table.Table, bool)
+	// SchemaNameAndTableNameByID is a pure memory operation.
+	SchemaNameAndTableNameByID(tableID int64) (schemaName, tableName ast.CIStr, ok bool)
```

Comment:
This is not used? `SchemaNameByTableID` is enough? or we can introduce this one and eliminate `SchemaNameByTableID`?

Reply:
Merged this two!